docs/instance_groups.md

```diff
@@ -186,3 +186,12 @@ Example: `kops delete ig morenodes`
 
 No rolling-update is needed (and note this is not currently graceful, so there may be interruptions to
 workloads where the pods are running on those nodes).
+
+## EBS Volume Optimization
+
+EBS-Optimized instances can be created by setting the following field:
```

Comment:
There are some instances types that cannot be created as EBS-Optimized instances. I created a cluster with an instance type that cannot be optimized, with the `rootVolumeOptimization` flag and this error was thrown by the task:

```
W0627 11:35:51.822386   28707 executor.go:109] error running task "LaunchConfiguration/master-us-east-1b.masters.test.k8s.us-east-1.example.com" (9m48s remaining to succeed): error creating AutoscalingLaunchConfiguration: ValidationError: Should specify either ephemeral volume, EBS volume, or no-device./dev/sda1.
```

We could maintain a list of the instances that cannot be EBS-Optimized instances, and throw a clearer error sooner. My concern is that the list gets out of sync when new instances types are released. But, it does appear that all the new generation instance types (except for T2) support EBS-Optimization, so maybe this list won't change much.

List of instance types that are not supported:

```
m1.small
m1.medium
c1.medium
cc2.8xlarge
cg1.4xlarge
m2.xlarge
cr1.8xlarge
i2.8xlarge
hi1.4xlarge
hs1.8xlarge
t1.micro
t2.nano
t2.micro
t2.small
t2.medium
t2.large
t2.xlarge
t2.2xlarge
m3.medium
m3.large
c3.large
c3.8xlarge
g2.8xlarge
r3.large
r3.8xlarge
```